src/app/pages/landing.page/landing.page.component.html

```diff
@@ -4,3 +4,17 @@ <h1>The future of irrigation is now</h1>
   <a href="" class="primary">Contact Us</a>
   <a href="" class="secondary">Learn More</a>
 </app-hero>
+
+@if (vm().productListVM.productItems.length > 0) {
+  <app-product>
+    <h2>Our Products</h2>
+    <app-product-list>
+      @for (product of vm().productListVM.productItems; track $index) {
```

Comment:
Isn't this confusing a bit? You call this productListVM while it contains the products as an array which is not part of the product-list component vm?